docs/deployment/deploy_with_existing_vineyard_cluster.md

```diff
@@ -1,8 +1,22 @@
-# Depoly with Existing Vineyard Cluster
+# Deploy with Existing Vineyard Cluster
 
 If you have already deployed a vineyard cluster, you can easily deploy GraphScope on the existing cluster and reuse the vineyard data such as graph with several GraphScope sessions. This will allow you to load a graph to the existing vineyard cluster and then reuse it with multiple GraphScope sessions, without needing to deploy a separate vineyard cluster for each session.
 
```

Comment:
"without needing to deploy a separate vineyard cluster for each session." => "without the need to deploy a separate vineyard cluster for each session."